engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnBehaviors.java

```diff
@@ -38,6 +38,8 @@ public interface BpmnBehaviors {
 
   BpmnJobBehavior jobBehavior();
 
+  BpmnSignalBehavior signalBehavior();
+
```

Comment:
👍 Great idea to add a signal behavior that can be reused for signal intermediate throw events.